tasks/configs.yml

```diff
@@ -23,6 +23,18 @@
     - restart csf
     - restart lfd
 
+# Uncomment some blocklists
+- name: enable blocklists 
+  replace: 
+    dest: /etc/csf/csf.blocklists
+    regexp: '^\#{{ item }}\|(\w+.*)?$'
+    replace: '{{ item }}|\1'
+  with_items: "{{ csf_blocklists_enable }}"
+  when: "csf_blocklists_enable is defined"
```

Comment:
Weird, the tests seem to be failing here but that _looks_ okay. Can you try changing it to be

```
when: csf_blocklists_enable is defined
```

Basically getting rid of the quotes?